src/socketio.quiver.js

```diff
@@ -315,103 +316,102 @@ QuiverSocialProvider.prototype.finishLogin_ = null;
  *
  * @override
  */
-QuiverSocialProvider.prototype.login = function(loginOpts, continuation) {
+QuiverSocialProvider.prototype.login = function(loginOpts) {
   if (this.isOnline_()) {
-    continuation(undefined, this.err("LOGIN_ALREADYONLINE"));
-    return;
+    return Promise.reject('LOGIN_ALREADYONLINE');
```

Comment:
We don't actually do this in other social providers - for example Firebase (https://github.com/freedomjs/freedom-social-firebase/blob/master/src/firebase-social-provider.js#L41) and GitHub (https://github.com/freedomjs/freedom-social-github/blob/master/src/github-social-provider.js#L786) both just reject with strings.  I could change this back to use this.err if you prefer it, but given that we don't actually check the error in uProxy and maybe just print it to the console I don't know that it makes much difference.